run/endpoints-v2-backend/gcloud_build_image

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
remove file and note this is mentioned in the linked endpoints + cloud run tutorial.